blog/dec-quiet-month-22.md

```diff
@@ -0,0 +1,41 @@
+---
+title: A Quiet Place (Dec'22)
+date: 2022-11-18T00:00:00.000Z
+authors:
+  name: plainekevin
+  url: 'https://github.com/PlaineKevin'
+  image_url: 'https://github.com/plainekevin.png?size=96'
+slug: a-quiet-place-22
+---
+
+The Electron project will pause for the month of December 2022, then return to full speed in January 2023.
+
+<div style={{width:"100%",height:"0",paddingBottom:"42%",position:"relative"}}>
+  <iframe src="https://giphy.com/embed/lbQQ1yeN4Z2wGm9q4d" width="480" height="270" frameBorder="0" class="giphy-embed" allowFullScreen></iframe>
+</div>
+<p><a href="https://giphy.com/gifs/marvelstudios-marvel-loki-studios-lbQQ1yeN4Z2wGm9q4d">via GIPHY</a></p>
+
+---
+
+## What will be the same in December
+
+1. Zero-day and other major security-related releases will be published as necessary. Security incidents should be reported via [SECURITY.md](https://github.com/electron/electron/tree/master/SECURITY.md).
```

Comment:
I think this should be changed for the new GitHub repo vulnerability thing (what @MarshallOfSound was talking about at the GitHub Universe panel).

Reply:
@MarshallOfSound updated the security.md to incorporate the new reporting feature so I think we should be set here